streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -362,9 +155,10 @@ public synchronized final TopologyBuilder addSource(final String name, final Str
      * @param topics the name of one or more Kafka topics that this source is to consume
      * @return this builder instance so methods can be chained together; never null
      */
-
-    public synchronized final TopologyBuilder addSource(final AutoOffsetReset offsetReset, final String name, final String... topics) {
-        return addSource(offsetReset, name, null, null, null, topics);
+    public synchronized final TopologyBuilder addSource(final AutoOffsetReset offsetReset,
+                                                        final String name, final String... topics) {
```

Comment:
nit: if we want to break into multi-lines better do that one parameter per line.